tests/e2e/common.go

```diff
@@ -193,7 +192,7 @@ func watchPods() {
 						fileName = filepath.Join(ResourceChangesISBServiceOutputPath, "pods", strings.Join([]string{pod.Name, ".yaml"}, ""))
 					case "mono-vertex", "mono-vertex-daemon":
 						fileName = filepath.Join(ResourceChangesMonoVertexOutputPath, "pods", strings.Join([]string{pod.Name, ".yaml"}, ""))
-					case "daemon", "vertex":
+					case "daemon", "vertex", "job":
 						fileName = filepath.Join(ResourceChangesPipelineOutputPath, "pods", strings.Join([]string{pod.Name, ".yaml"}, ""))
```

Comment:
how about having a default case if it doesn't match any of the above that does a `continue`?